main.go

```diff
@@ -135,29 +133,43 @@ func main() {
 		}
 	}
 
-	// Generate the review and parse the assistant's recommendation
-	review, reviewComments, action, err := generateReviewWithAssistant(pr, files)
-	if err != nil {
-		fmt.Printf("Error generating review: %v\n", err)
-		os.Exit(1)
-	}
+	var review string
+	var reviewComments []*github.DraftReviewComment
+	var action string
 
-	// Output the generated review
-	log.Println("------- Generated Review:")
-	log.Println(review)
-	log.Println("------- File comments:")
-	for _, comment := range reviewComments {
-		log.Printf("File: %s, Line: %d\nComment: %s\n", *comment.Path, *comment.Position, *comment.Body)
+	// if there is no review, or we are forcing a new one
+	if savedReview == nil || (forcedry != nil && *forcedry) {
```

Comment:
This condition could be confusing; consider renaming `forcedry` to something more descriptive like `forceDryRun`.